frontend/packages/integration-tests-cypress/reporter-config.json

```diff
@@ -9,6 +9,7 @@
     "reportFilename": "cypress_report",
     "overwrite": false,
     "html": false,
-    "json": true
+    "json": true,
+    "quiet" : true
```

Comment:
I would remove `quiet` here.

Reply:
Why?  Do we need the `[mochawesome] Report JSON saved to /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/gui_test_screenshots/cypress_report_004.json` outputs?

Reply:
Why do we want to suppress that? It could help debugging if there is ever an issue generating the report.

Reply:
ok, reverted.